property_tests/tests/stable_b_tree_map/test/get.ts

```diff
@@ -46,9 +46,13 @@ export function GetTestArb(
 function generateBody(stableBTreeMapName: string): string {
     return /*TS*/ `
         const result = ${stableBTreeMapName}.get(param0);
-        if (result === null) {
+        const containsKey = ${stableBTreeMapName}.containsKey(param0); // For situations where the stored value is literally null
+        console.log("contains key?", containsKey)
```

Comment:
Can we remove these logs?

Reply:
yes they were only there because this wasn't actually ready to be reviewed